common/app/views/fragments/commentBox.scala.html

```diff
@@ -21,13 +21,24 @@
         </div>
     </div>
 
+    <div class="d-comment-box__onboarding">
+        <h3>Welcome, you’re about to make your first comment!</h3>
```

Comment:
@mchv this is now done!

<img width="644" alt="screen shot 2016-04-11 at 11 08 24" src="https://cloud.githubusercontent.com/assets/858402/14424079/0436af3e-ffd7-11e5-9d1d-e12c5a447e55.png">